platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/reconnect/ReconnectController.java

```diff
@@ -54,13 +54,13 @@ public class ReconnectController implements Runnable {
      * @param helper
      * 		executes phases of a reconnect
      * @param startChatter
-     * 		starts chatter if previously suspended
+     * 		starts gossip if previously suspended
      */
     public ReconnectController(
             final ThreadManager threadManager, final ReconnectHelper helper, final Runnable startChatter) {
```

Comment:
You renamed the member, but missed the parameter `startChatter`

Reply:
fixed